CONST_Makefile

```diff
@@ -89,7 +89,8 @@ NGEO_LIBS_JS_FILES += \
 	node_modules/bootstrap/dist/js/bootstrap.min.js \
 	node_modules/proj4/dist/proj4.js \
 	node_modules/d3/d3.min.js \
-	node_modules/typeahead.js/dist/typeahead.bundle.min.js
+	node_modules/typeahead.js/dist/typeahead.bundle.min.js \
+        node_modules/fuse.js/src/fuse.min.js 
```

Comment:
`NGEO_LIBS_JS_FILES += node_modules/fuse.js/src/fuse.min.js ` in the `geoportalv3.mk` file